server/datastore/mysql/packs.go

```diff
@@ -136,6 +136,7 @@ func (ds *Datastore) GetPackSpecs(ctx context.Context) ([]*fleet.PackSpec, error
 
 		// Load targets
 		for _, spec := range specs {
+			spec := spec
```

Comment:
Yeah those were fine since we don't store the address (it gets used inside the loop before the next iteration), but makes sense that it's a bit much to ask of the lint check to go to that level of analysis.